weave/trace_server/migrations/002_add_deleted_at.up.sql

```diff
@@ -5,51 +5,6 @@ This migration adds:
         - the object_versions_deduped and calls_merged_view views
 */
 
-ALTER TABLE object_versions
```

Comment:
IGNORE: will pull out of PR, this is just what I had to do to get my dev setup working